modules/migration-configuring-cors-3.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// migration/migrating_openshift_3_to_4/migrating-openshift-3-to-4.adoc
+[id='migration-configuring-cors-3_{context}']
+= Configuring cross-origin resource sharing on an {product-title} 3 cluster
+
+You must configure cross-origin resource sharing on the {product-title} 3 cluster to enable communication between the CAM tool on the target cluster and the source cluster's API server.
+
+.Procedure
+
+. Log in to the {product-title} 4 cluster.
+. Obtain your CAM tool URL:
```

Comment:
Comments from Marco (he left them originally on the CORS 4 configuration page)

I would change:
"Obtain the CAM tool URL:" for "Obtain the CAM tool CORS configuration line:"

I actually thought I was getting a broken REGEX line before realizing it wasn't suppose to return the route, but the exact line config you need with the special characters.

Reply:
Yes, that does make a difference. I think that it would be a good idea for me to remove the note explaining the syntax because the user just has to copy the CORS config line.